includes/Modules/Sign_In_With_Google.php

```diff
@@ -137,8 +253,9 @@ private function render_signin_button() {
 	google.accounts.id.initialize({
 		client_id: '<?php echo esc_js( $settings['clientID'] ); ?>',
 		login_uri: '<?php echo esc_js( $redirect_url ); ?>',
+		ux_mode: 'redirect',
```

Comment:
It needs to be redirected instead of being opened in popup, in order to work correctly